roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -135,15 +135,9 @@ supported_os_distributions:
 
 # Extending some distributions into the redhat os family
 redhat_os_family_extensions:
-  - "Kylin Linux Advanced Server"
-  - "openEuler"
   - "UnionTech"
```

Comment:
Btw this seems to be also added now: https://github.com/ansible/ansible/commit/34e60c0a7ab67d5d7dbf133de728aeb68398dd02

Reply:
Yeah but there is several instances of Uos , debian based (Deepin actually) and Red Hat apparently: see the [thread in slack](https://kubernetes.slack.com/archives/CDQSC941H/p1710151485346759)